class_person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, parent_permission: true, name: 'Unknow')
+    @id = id
```

Comment:
- [ ] The id parameter would not be requested to create a person. It will be nice if we can generate random numbers for it.` @id = Random.rand(1..1000)`

Reply:
Bro, why are you trying to complicate the project? This random is not required, and the ID is not required either, but It's not a problem if I add it